lib/screens/price_screen.dart

```diff
@@ -225,56 +222,42 @@ class _PriceScreenState extends State<PriceScreen> {
                       ),
                       _noOfEditionsFieldError.isNotEmpty
                           ? Padding(
-                        padding: EdgeInsets.only(left: 10.w, right: 10.w, top: 2.h),
-                        child: Text(
-                          _noOfEditionsFieldError,
-                          style: TextStyle(
-                            fontSize: 12.sp,
-                            color: Colors.red,
-                          ),
-                        ),
-                      )
+                              padding: EdgeInsets.only(left: 10.w, right: 10.w, top: 2.h),
+                              child: Text(
+                                _noOfEditionsFieldError,
+                                style: TextStyle(
+                                  fontSize: 12.sp,
+                                  color: Colors.red,
+                                ),
+                              ),
+                            )
                           : const SizedBox.shrink(),
                       Text(
                         "${NumberFormat.decimalPattern().format(kMaxEdition)} $kMaxText",
                         style: TextStyle(color: EaselAppTheme.kLightPurple, fontSize: 14.sp, fontWeight: FontWeight.w800),
                       ),
-
                     ],
                   ),
                 ),
                 VerticalSpace(20.h),
-
                 Align(
                   alignment: Alignment.centerRight,
                   child: PylonsButton(
                     onPressed: () async {
                       FocusScope.of(context).unfocus();
                       if (_formKey.currentState!.validate()) {
-                        if(!provider.isFreeDrop){
-                          if (_royaltiesFieldError.isEmpty &&
-                              _noOfEditionsFieldError.isEmpty &&
-                              _priceFieldError.isEmpty) {
-                            widget.controller
-                                .nextPage(duration: const Duration(milliseconds: 300), curve: Curves.easeIn);
-                          }
+                        if (_royaltiesFieldError.isEmpty && _noOfEditionsFieldError.isEmpty && _priceFieldError.isEmpty) {
```

Comment:
Make this line a method please. _royaltiesFieldError.isEmpty && _noOfEditionsFieldError.isEmpty && _priceFieldError.isEmpty